cloudinit/config/cc_install_hotplug.py

```diff
@@ -49,6 +49,7 @@
         """
         ),
     ],
+    "activate_by_schema_keys": [],
```

Comment:
good as we potentially need to uninstall hotplug config files if not enabled or available regardless of config